modules/gathering-data-specific-features.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+// * cnv/cnv_users_guide/cnv-collecting-cnv-data.adoc
+// Dependency: modules/cnv-document-attributes.adoc
+
+[id="gathering-data-specific-features_{context}"]
+= Gathering data about specific features
+
+You can gather debugging information about specific features by using the
+`oc adm must-gather` CLI command with the `--image` or `--image-stream` argument.
+The `must-gather` tool supports multiple images, so you can gather data about
+more than one feature by running a single command.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
+
+. Run the `oc adm must-gather` command with one or more `--image` or `--image-stream`
+arguments. For example, the following command gathers both the default cluster
+data and information specific to {CNVProductName}:
++
+----
+$ oc adm must-gather \
+ --image-stream=openshift/must-gather \ <1>
+ --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8 <2>
```

Comment:
cnv-must-gather image is invalid/doesn't exist. After completing the openshift part of the must-gather, the command  fails with this error:
`error: gather did not start for pod must-gather-f9tvl: unable to pull image: ErrImagePull: rpc error: code = Unknown desc = Error reading manifest latest in registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8: unknown: Not Found`